Cargo.toml

```diff
@@ -3,102 +3,88 @@ name = "bevy"
 version = "0.3.0"
 edition = "2018"
 authors = [
-    "Bevy Contributors <bevyengine@gmail.com>",
-    "Carter Anderson <mcanders1@gmail.com>",
+  "Bevy Contributors <bevyengine@gmail.com>",
+  "Carter Anderson <mcanders1@gmail.com>",
 ]
+categories = ["game-engines", "graphics", "gui", "rendering"]
 description = "A refreshingly simple data-driven game engine and app framework"
+exclude = ["assets/**/*", "tools/**/*", ".github/**/*", "crates/**/*"]
 homepage = "https://bevyengine.org"
-repository = "https://github.com/bevyengine/bevy"
-license = "MIT"
 keywords = ["game", "engine", "gamedev", "graphics", "bevy"]
-categories = ["game-engines", "graphics", "gui", "rendering"]
-readme = "README.md"
```

Comment:
The readme key is gone.

Reply:
Bawhaha nice catch. We definitely need that!